doc/source/locale/zh_CN/LC_MESSAGES/user_guide/deployment_slurm.po

```diff
@@ -149,77 +150,73 @@ msgid "Obtain the nodes"
 msgstr "获取节点"
 
 #: ../../source/user_guide/deployment_slurm.rst:74
-msgid "Next, we'll want to obtain the head node and the IP address."
-msgstr "接下来，你需要获取头结点和其 IP 地址。"
+msgid "Next, we'll want to obtain all the allocated compute nodes."
```

Comment:
Could you please tell us that have you confirmed that these are hostnames rather than ip address on SLURM?

Reply:
Yes, I am sure that every compute node has a hostname, as the system admins must configure the hostname so that users can get the node name. 
I test these settings in my environments. 
And note that this is only a suggestion, as different clusters have their configurations. We cannot cover all cases. Users can refer to this setting and write their scripts.